devel/jreleaser/Portfile

```diff
@@ -4,54 +4,56 @@ PortSystem       1.0
 PortGroup        java 1.0
 
 name             jreleaser
-version          0.8.0
+version          0.9.0
 revision         0
-                 
+
 categories       devel java
-license          Apache-2
+license          Apache-2.0
 maintainers      @aalmiray
```

Comment:
Please make this port also `openmaintainer` unless you have a specific reason not to:

```
maintainers      @aalmiray openmaintainer
```

Without `openmaintainer`, it is implied that you and _only_ you are allowed to make changes or updates to this port - which may be required in some circumstances, but something we try to avoid in most cases.

Reply:
https://guide.macports.org/chunked/project.update-policies.html

Reply:
I see. This port file is automatically generated using the JReleaser tool itself. The current workflow is to publish the portfile first to https://github.com/jreleaser/jreleaser-macports, refresh a personal fork of this repository and manually submit a PR. Once a release has been posted there should be no need to update the portfile, until the next release.

Would that qualify as exempting this portfile from adding `openmaintainer`?

Reply:
Herby, any concerns with merging this update, and deferring the opennaintainer conversation for later?

Reply:
> 

Yes that does sound reasonable. @mascguy, no concerns.

Reply:
Sounds good, I'll go ahead and merge. It's definitely a reasonable question though!

Reply:
Thank you. I must confess I was not aware of the `openmaintainer` tag until today. I could certainly add it but the chances of this port being updated outside of its automated flow _should_ be small (yet never zero I suppose).